backend/src/main/kotlin/cz/loono/backend/notification/NotificationDefinition.kt

```diff
@@ -0,0 +1,195 @@
+package cz.loono.backend.notification
+
+import cz.loono.backend.api.dto.BadgeTypeDto
+import cz.loono.backend.api.dto.ExaminationTypeDto
+import cz.loono.backend.api.dto.SexDto
+import cz.loono.backend.api.exception.LoonoBackendException
+import cz.loono.backend.db.model.Account
+import org.springframework.http.HttpStatus
+
+object NotificationDefinition {
+
+    private const val ONESIGNAL_APP_ID = "234d9f26-44c2-4752-b2d3-24bd93059267"
+    private const val MORNING_TIME_TO_NOTIFY = "8:00AM"
+    private const val EVENING_TIME_TO_NOTIFY = "6:00PM"
+
+    fun getPreventionNotification(accounts: Set<Account>): PushNotification {
+        val name = "Prevevention notification"
+        val title = "Hola, hola, prevence volá!"
+        val text = "Mrkni, na které preventivní prohlídky se objednat."
+        return PushNotification(
+            appId = ONESIGNAL_APP_ID,
+            name = name,
+            headings = MultipleLangString(cs = title, en = title),
+            contents = MultipleLangString(cs = text, en = text),
+            includeExternalUserIds = accounts.map { it.uid },
+            scheduleTimeOfDay = MORNING_TIME_TO_NOTIFY,
+            data = NotificationData(screen = "main")
+        )
+    }
+
+    fun getCompletionNotification(
+        accounts: Set<Account>,
+        time: String,
+        examinationTypeDto: ExaminationTypeDto
+    ): PushNotification {
+        val name = "Complete checkup notification"
+        val title = "Byl/a jsi na prohlídce?"
+        val text = "Potvrď preventivní prohlídku v aplikaci a získej odměnu."
```

Comment:
Nedáme to niekde do .properties file? Náhodne v kóde je cesta k neudržiteľnosti.